### PR TITLE
add weather app task

### PR DESCRIPTION
**Pull request by AMashoshyna**

### CODE REVIEW

File: tasks/language_basics_2.md [hunk 1]
```diff
@@ -2,12 +2,37 @@
...
+
+Example:
+
+```bash
+my-weather-app --l=London --r=week
+```
+This request will show weather forecast for next week in London
+
+```bash
+my-weather-app r=week
```
Comment (lempiy) [inline tasks/language_basics_2.md]:
```bash
my-weather-app r=week
```
```
my-weather-app --r=week
```

**Review by lempiy — CHANGES_REQUESTED** [review]

File: tasks/language_basics_2.md [hunk 2]
```diff
@@ -2,12 +2,37 @@
...
+All cities I have entered will be stored permanently to file. If I start the program without providing any location, last successful query will be repeated.
 
-1. :vertical_traffic_light:__Node.js__:vertical_traffic_light: [Part II](js_basics_2.md)
-1. :vertical_traffic_light:__Golang__:vertical_traffic_light: [Part II](golang_basics_2.md)
-1. :vertical_traffic_light:__Python__:vertical_traffic_light: [Part II](python_basics_2.md)
+Use Openweather API for requests https://openweathermap.org/api.
 
+City and forecast range should be provided as command-line arguments:  
+`l` for "location": enter city name. Optional. Default value - last successful location name followed by successful query.  
+`r` for "range": supported values: “week” and “day”. Optional. Default value “day"
```
Comment (intrueder) [inline tasks/language_basics_2.md]: I'd go with `-l`, `--location`, `-r` and `--range` parameters

File: tasks/language_basics_2.md [hunk 3]
```diff
@@ -2,12 +2,37 @@
...
+
+```bash
+my-weather-app --l=London --r=week
+```
+This request will show weather forecast for next week in London
+
+```bash
+my-weather-app r=week
+```
+This request triggers prompt “please enter location”
```
Comment (intrueder) [inline tasks/language_basics_2.md]: Don't forget that location is optional. And prompt should be triggered only when default value (last successful call) is not available.

File: tasks/language_basics_2.md [hunk 4]
```diff
@@ -2,12 +2,37 @@
...
-1. :vertical_traffic_light:__Python__:vertical_traffic_light: [Part II](python_basics_2.md)
+Use Openweather API for requests https://openweathermap.org/api.
 
+City and forecast range should be provided as command-line arguments:  
+`l` for "location": enter city name. Optional. Default value - last successful location name followed by successful query.  
+`r` for "range": supported values: “week” and “day”. Optional. Default value “day"
+
+When called without arguments, the app gets weather for successful location. If no location was chosen, textual message “Please enter the location” will be printed.
+
+Example:
```
Comment (intrueder) [inline tasks/language_basics_2.md]: Need at least one more example to cover the case with default range:

> ```bash
> my-weather-app --l=London
> ```
> This request will show weather forecast for next day in London

**Review by intrueder — CHANGES_REQUESTED** [review]

**Pull request merged**